src/LIVR.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace uMotif;
+
+use Validator\LIVR as LIVRBase;
+
+class LIVR extends LIVRBase
+{
+    private static $DEFAULT_RULES = array(
```

Comment:
Do you need all of this? The reigster rules method merges the rules you give it with the rules it already knows about. When you call the parent constructor it should register all of these rules for you, so all you need to do is call register rules again after the parent constructor call and pass in the new custom rule.